### DISCUSSION TITLE
Disabling Overdrag

### DISCUSSION THREAD

**Discussion by mdyszynski** [lodev09/react-native-true-sheet#150] [Ideas]
Hi,

First of all, love TrueSheet. Works amazing. I'm wondering if there is a way to disable overdrag if there is only one height? I can't seem to find anything in the docs about that.

**Comment by lodev09** [comment]
There's no straight config that we can set for this, unfortunately. My guess is to mess around with the gesture in IOS. I'm open for suggestions though -- overdrag is kinda default behavior in IOS :/

Moving to discussions
